src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -292,11 +293,22 @@ public String getMapKeyAccessorName(TypeModel keyType, String key) {
   }
 
   @Override
-  public String getFormattedPrintArgName(TypeModel type, String variable, List<String> accessors) {
+  public String getFormattedPrintArgName(
+      ImportTypeTable typeTable, TypeModel type, String variable, List<String> accessors) {
+    Preconditions.checkArgument(
+        !type.isRepeated() && !type.isMap(),
+        "Expected non-repeated fields in print statement, found %s",
```

Comment:
nit: would s/non-repeated/scalar/ be more accurate?

Reply:
It seems protobuf only refers to string, numbers, bytes and bools as scalar fields.
But the field could be a nested message or an enum here.